c2corg_ui/templates/helpers/edit.html

```diff
@@ -105,11 +105,6 @@ <h3 class="heading show-phone" translate>summary</h3>
         <input type="text" name="message" ng-model="saveCtrl.message" class="form-control" placeholder="{{'Short description of the applied changes' | translate}}" />
       </div>
       % endif
-      ## QUALITY
-      <div class="full comment" ng-class="{'has-success': saveCtrl.quality}">
-        <label translate>quality</label>
-        <select ng-model="saveCtrl.quality" ng-options="type as saveCtrl.langService.translate(type) for type in ${quality_types}" class="form-control"></select>
-      </div>
```

Comment:
Are you sure we really want to skip all control on the final quality choice?
That looks a bit too radical too me. The quality of a document also depends on the content itself: if you have all the fields set but with a silly or poor content, the quality will never be excellent!
I think we should keep the manual selector, at least the time needed to make sure the automatic setting actually works.

Reply:
This decision comes from a vote: cf. https://forum.camptocamp.org/t/vote-sur-le-calcul-du-critere-de-completude/223776/26

Reply:
I do think this is not safe to introduce such a radical change without a significative period of testing, to make sure the auto computation actually works as expected.

In addition if the goal was to have the quality set with no human interaction, your changes should have been done in the API, not in the UI. By the way what happens if someone makes changes through the mobile app or an other API client?

And as I said, it seems to me your computation only use quantitative criterias, and not really qualitative ones. For instance if you have a topo full of "blabla" it is definitely not as good as if you have a shorter but more usable description.

I remove my change request but am definitely reluctant to give my approval.